pkg/asset/installconfig/aws/subnet.go

```diff
@@ -23,13 +26,47 @@ type Subnet struct {
 
 	// CIDR is the subnet's CIDR block.
 	CIDR string
+
+	// ZoneType is the type of subnet's availability zone.
+	// The valid values are availability-zone and local-zone.
+	ZoneType string
+
+	// ZoneGroup is the group of subnet's availability zone is part of.
```

Comment:
Reword to make this clear.

Reply:
[Done](patch)

[patch]:https://github.com/openshift/installer/compare/ee094569c1e9b59a55d694f6d83811e57c1c8b87..95bc39b33543d489b20c2da0cf3e38ef6f10dc3f